src/components/ToggleBot.js

```diff
@@ -1,23 +1,50 @@
-import { useVariableValue } from '@devcycle/react-client-sdk';
-import togglebot from '../togglebot.png';
-import togglebotWink from '../togglebot-wink.png';
+import { useDevCycleClient, useVariableValue } from '@devcycle/react-client-sdk';
 import classNames from 'classnames';
 
 function ToggleBot() {
   /**
    * The useVariableValue hook will return the current value of a variable.
    * If no value is defined for the current user, the default value will be returned.
    */
+  const devcycleClient = useDevCycleClient()
   const shouldWink = useVariableValue('togglebot-wink', false)
   const spinSpeed = useVariableValue('togglebot-speed', 'off')
 
+  const variation = devcycleClient.config.features['hello-togglebot']?.variationName ?? 'Default'
+
   return (
-    <img
-      src={shouldWink ? togglebotWink : togglebot}
-      className={classNames(['App-logo', `spin-${spinSpeed}`])}
-      alt="togglebot"
-    />
+    <div className="App-content">
+      <div className="ToggleBot-message">
+        "{getMessage(spinSpeed)}"
+      </div>
+      <img
+        src={getImageSource(spinSpeed, shouldWink)}
+        className={classNames(['ToggleBot-logo', `spin-${spinSpeed}`])}
+        alt="togglebot"
+      />
+      <div className="ToggleBot-variation">
+        Serving Variation: <b>"{variation}"</b>
+      </div>
+    </div>
   );
 }
 
+const getMessage = (spinSpeed) => {
+  switch (spinSpeed) {
+    case 'fast':
+      return 'Gotta go fast!'
+    case 'off-axis':
+      return '...I\'m gonna be sick...'
+    case 'surprise':
+      return 'Unicorn!'
+    default:
+      return 'Hello! Nice to meet you.'
```

Comment:
i think this is missing the 'slow' case?

Reply:
Yeah, I realized the designs didn't include a message specifically for slow — unless we're thinking we'll use "WEEEE" for that? That was the only one in the designs that wasn't labelled

Reply:
oh true i was assuming "WEEEE" was for slow lol, i didn't realize it wasn't labelled